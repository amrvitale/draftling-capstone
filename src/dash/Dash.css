body, html {
    height: 100%;
    margin: 0;
}

.deskHero {
    background-image: url(./mydesk.jpg);
    background-position: center; /* Center the image */
    background-repeat: no-repeat; /* Do not repeat the image */
    background-size: cover; /* Resize the background image to cover the entire container */
    min-height: 350px;
}

.myDesk {
    color: #35322a;    
    font-family: 'Playfair Display', 'Georgia', serif;
    font-size: 120px;
    font-weight: 900;
}

.start {
    font-size: 40px;
    color: whitesmoke;
}
p {
    font-size: 30px;
    color: whitesmoke;
}

.postNew {
height: 80px;
}

.viewMyDraftlings{
height: 80px;
}