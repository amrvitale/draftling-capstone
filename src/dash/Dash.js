import React from 'react';
import './Dash.css';
import { Link } from 'react-router-dom';
 

class Dash extends React.Component {
    render() {
        return (
            <div className="draftlingDeskPage">
                <div className="deskHero">
                    <h1 className="myDesk">My Draftlings Desk</h1>
                    <br />
                    <br />
                </div>
                    <br />
                    <br />
                    <br />
                    <h2 className="start">Let's get started!</h2>
                    <p>Welcome to your desk, your own personal workspace at Draftling. <br />Here you can create a brand new draftling, or view your existing draftlings to edit or publish to the Draftling database.</p>
                    <br />
                    <br />
                    <Link to='/postdraftling'>
                        <button type="submit" className="postNew">Post New Draftling</button>
                    </Link>

                    <br/>
                    <br/>

                    <Link to='/mydraftlings'>
                        <button type="submit" className="viewMyDraftlings">View My Draftlings</button>
                    </Link>
            </div>
        );
    }
}
export default Dash;