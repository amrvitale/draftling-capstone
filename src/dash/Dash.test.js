import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Dash from './Dash'

describe (`Dash component`, () => {
    it('renders a .Dash by default', () => {
        const wrapper = shallow(<Dash />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })

    
})