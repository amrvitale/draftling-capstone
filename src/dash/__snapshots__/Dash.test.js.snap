// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dash component renders a .Dash by default 1`] = `
<div
  className="draftlingDash"
>
  <h1>
    My Draftling Dashboard
  </h1>
  <Link
    to="/postdraftling"
  >
    <button
      type="submit"
    >
      Post New Draftling
    </button>
  </Link>
  <br />
  <br />
  <Link
    to="/mydraftlings"
  >
    <button
      type="submit"
    >
      View My Draftlings
    </button>
  </Link>
</div>
`;
